content/docs/1.8.0/advanced-resources/security/volume-encryption.md

```diff
@@ -107,7 +107,12 @@ StorageClass parameters are needed to enable online expansion:
 - `csi.storage.k8s.io/node-expand-secret-name`
 - `csi.storage.k8s.io/node-expand-secret-namespace`
 
+# V2 Volume
+
+- Longhorn will create a `dm` device named `<volume-name>-encrypted` for encrypted V2 volume provided to the workload. This is because a V2 volume requires a `dm` device named `<volume-name>` to manage I/O operations, such as during a V2 volume upgrade.
+- The v2 volume is not supporting the expansion in v1.8.0.
```

Comment:
Move it to the above `# Filesystem Expansion`. Using `Notice` instead.

Reply:
Moved. Thanks.